wordpress-mobile/WordPress-iOS
# Add accessibility information to toolbar items.

Fixes #6945

To test:
 - Start the app on the simulator
 - Open the aztec editor
 - Using the Accessibility Inspector (Xcode -> Open Developer Tools) check that all toolbar items have the correct information


Needs review: @jleandroperez
